client/signup/config/flows.js

```diff
@@ -280,7 +280,7 @@ function filterFlowName( flowName ) {
 	return flowName;
 }
 
-function filterDestination( destination, dependencies, flowName ) {
+function filterDestination( destination ) {
 	return destination;
 }
 
```

Comment:
Right, this change is here only because because this file wasn't under linter control before :)
`identity` seems to be a good idea here.